### ISSUE TITLE
[문의] 세가지 프로젝트가 branch로 구별이 되어있는것 같습니다. 

### ISSUE THREAD

**Issue by miking38** [npc6017/Slang-Statistics#1]
RestAPI-deep-learning
RestAPI-middle
android 

세개의 프로젝트가 각각의 브랜치에 저장이 되어있는데, 이렇게 만드신 이유가 궁금합니다. 
나중에 master branch 로 합쳐지는 것일까요?

**Comment by Ji-Ha** [comment]
맞습니다. 프로젝트가 작다고 생각해서 브랜치로 나눠서 작업을 하고, 최종적으로는 master branch로 merge하려고 했습니다.
그리고 이 repository는 겨울방학에 진행했던 작업이고, 지금 진행중인 것은 저희가 private로 진행하고 있어서 보실 수 없었던 것 같습니다! (하지만 branch를 나눠서 하는 상황은 똑같습니다.) 진행중인 프로젝트 역시 public으로 바꿔놓을테니 한번 봐주시고 많은 조언 부탁드립니다! 감사합니다ㅎㅎ
